utils/autogen/run_si.py

```diff
@@ -13,8 +13,8 @@
 
 #An example to submit to the Taub campus cluster.
 #Change these 
-directory="/home/lkwagner/project/tmp/" #the trailing slash is important!
-username="lkwagner"
+directory="/projects/erg/jaschil2/QMCDB/" #the trailing slash is important!
```

Comment:
We should probably just put in placeholders here, and not commit our names back and forth.